hivemind_listener/transformers.py

```
@@ -0,0 +1,217 @@
+
+    def load_plugins(self):
+        for plug_name, plug in find_dialog_transformer_plugins().items():
+            if plug_name in self.config:
+                # if disabled skip it
+                if not self.config[plug_name].get("active", True):
+                    continue
+                try:
+                    self.loaded_plugins[plug_name] = plug(config=self.config[plug_name])
+                    self.loaded_plugins[plug_name].bind(self.bus)
+                    LOG.info(f"loaded audio transformer plugin: {plug_name}")
+                except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)